web3deployment/oasisReserveDeployer.js

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env node
+
+const Web3 = require("web3");
+const fs = require("fs");
+const path = require('path');
+const RLP = require('rlp');
+const BigNumber = require('bignumber.js')
+
+process.on('unhandledRejection', console.error.bind(console))
+
+//current run command: node web3deployment/oasisReserveDeployer.js --gas-price-gwei 3 --rpc-url https://mainnet.infura.io 
+const { gasPriceGwei, printPrivateKey, rpcUrl, signedTxOutput, dontSendTx, chainId: chainIdInput } = require('yargs')
+    .usage('Usage: $0 --gas-price-gwei [gwei] --print-private-key [bool] --rpc-url [url] --signed-tx-output [path] --dont-send-tx [bool] --chain-id')
+    .demandOption(['gasPriceGwei', 'rpcUrl'])
+    .boolean('printPrivateKey')
+    .boolean('dontSendTx')
+    .argv;
+const web3 = new Web3(new Web3.providers.HttpProvider(rpcUrl));
+const solc = require('solc')
+
+const rand = web3.utils.randomHex(7);
+
+const privateKey = web3.utils.sha3("in joy we trust" + rand);
+console.log("privateKey", privateKey);
+
+if (printPrivateKey) {
+  let path = "privatekey_"  + web3.utils.randomHex(7) + ".txt";
+  fs.writeFileSync(path, privateKey, function(err) {
+      if(err) {
+          return console.log(err);
+      }
+  });
+}
+const account = web3.eth.accounts.privateKeyToAccount(privateKey);
+const sender = account.address;
+const gasPrice = BigNumber(gasPriceGwei).mul(10 ** 9);
+const signedTxs = [];
+let nonce;
+let chainId = chainIdInput;
+
+console.log("from",sender);
+
+async function sendTx(txObject) {
+  const txTo = txObject._parent.options.address;
```

Comment:
all this function is written with 2 space tabs.
Should be 4.
Yaron should configure his atom editor. 
"coming soon"